homeassistant/components/comelit/config_flow.py

```diff
@@ -163,6 +163,43 @@ async def async_step_reauth_confirm(
             errors=errors,
         )
 
+    async def async_step_reconfigure(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """Handle reconfiguration of the device."""
+        reconfigure_entry = self._get_reconfigure_entry()
+        if not user_input:
+            return self.async_show_form(
+                step_id="reconfigure", data_schema=STEP_RECONFIGURE
+            )
+
+        updated_host = user_input[CONF_HOST]
+
+        if reconfigure_entry.data[CONF_HOST] != updated_host:
+            self._async_abort_entries_match({CONF_HOST: updated_host})
```

Comment:
This isn't needed anymore, we now have added an exception to `self._async_abort_entries_match` which makes sure we ignore the current entry

Reply:
Done